tests/connection/broker.cpp

```diff
@@ -0,0 +1,71 @@
+
+/* Copyright (c) 2014-2015, Human Brain Project
+ *                          Stefan.Eilemann@epfl.ch
+ */
+
+#define BOOST_TEST_MODULE zeq_connection_broker
+
+#include "../broker.h"
+#include <zeq/connection/broker.h>
+
+#include <lunchbox/sleep.h>
+#include <lunchbox/thread.h>
+#include <lunchbox/servus.h>
+#include <boost/bind.hpp>
+
+using boost::lexical_cast;
+
+const unsigned short port = (lunchbox::RNG().get<uint16_t>() % 60000) + 1024;
+
+class Service : public lunchbox::Thread
```

Comment:
A service subscribes to request events to publish the reply data. What is fishy for you?